db/schema.rb

```diff
@@ -268,10 +268,10 @@
   create_table "broadcasts", id: :serial, force: :cascade do |t|
     t.boolean "active", default: false
     t.text "body_markdown"
-    t.datetime "created_at"
     t.text "processed_html"
     t.string "title"
     t.string "type_of"
+    t.datetime "created_at"
```

Comment:
I'm not exactly sure why this moved for me 🤔

Reply:
My guess is it's related to https://github.com/thepracticaldev/dev.to/pull/8181 ?